### DISCUSSION TITLE
Immediately deleting a blank draft gets Authorization Error

### DISCUSSION THREAD

**Discussion by ben-gladeye** [outline/outline#3568] [Self Hosting]
Hello!

We ran into an `Authorization Error` recently when deleting a new doc immediately after creating it.

Reproduction:
1. Click + New doc
2. Without adding a title or content, immediately click delete & `I'm sure`
3. Meet Authorization Error

Observing the Network log I can see the call to `documents.delete` runs twice, first call is successful and the second is the `Authorization Error`.

Outline version: `v0.63.0` Self hosted
`Ubuntu 20.04`

<img width="652" alt="image" src="https://user-images.githubusercontent.com/72585149/169747104-77a315a0-2400-40aa-a403-a7abba388332.png">

**Comment by tommoor** [comment]
This only shows up in logs but not in the UI, yep?

**Reply by ben-gladeye** [reply]
We do get the error in the UI, how can we suppress/log that instead btw?

![image](https://user-images.githubusercontent.com/72585149/169902972-76ff4174-2afd-4b87-ba2b-b3aba8126dd6.png)

**Reply by tommoor** [reply]
This is a dev error, you're running in development mode. Need NODE_ENV=production

**Reply by ben-gladeye** [reply]
Ah of course it is, will flick that over now, thanks!

**Reply by dl-lim** [reply]
I can confirm I'm also seeing this same error on 0.64.4. Is there an issue for this already?

**Reply by tommoor** [reply]
It's not really considered an issue at this time
